### PR TITLE
Trying to clarify documentation

### PR DESCRIPTION
**Pull request by davidweichiang**

### CODE REVIEW

File: anthologize/README.md [hunk 1]
```diff
@@ -22,48 +20,49 @@ You'll need Python >=3.5 and the Python packages `latexcodec` and `pybtex`; thes
 
 ## 2. Assembling the data
 
-ACLPUB is written primarily with Softconf's STARTv2 system in mind.
+If you are using:
+
+- STARTv2: follow the instructions in Step 2(a), then 2(c).
+- EasyChair: follow the instructions in Step 2(b), then 2(a) and 2(c).
+- Another system: you will need to manually arrange your files into the layout described in Step 2(c).
```
Comment (davidweichiang) [inline anthologize/README.md]: Or should users of other systems directly create the file format in Step 3?

File: anthologize/README.md [hunk 2]
```diff
@@ -188,19 +190,23 @@ where `<x>` is the one-letter code of the conference (e.g., `N` for NAACL) and `
 
 ## 4. Package and send the Anthology directory
 
-Package up the anthology directory and send it to your book chair.
-If you are the book chair, please contact the Anthology Director.
+Package up the anthology directory:
 
     tar czvf <conference>_anthology.tgz anthology
 
+Send the resulting file (`<conference>_anthology.tgz`), or a link to it, by email to the Anthology Director.
```
Comment (davidweichiang) [inline anthologize/README.md]: Is this right?

File: anthologize/README.md [hunk 3]
```diff
@@ -209,15 +215,16 @@ Current types supported by the Anthology are:
 + Software
 + Attachment (this is for generic attachments; best to be specific and avoid using this when possible)
 
-Videos are also accepted but only as valid hyperlinks.
+Videos are also accepted, but only as valid hyperlinks.
```
Comment (davidweichiang) [inline anthologize/README.md]: What is the attachment type for videos?

File: anthologize/README.md [hunk 4]
```diff
@@ -209,15 +215,16 @@ Current types supported by the Anthology are:
...
+or to a valid containing "home page" for the resource. They should be
 maintained in perpetuity, but reasonably maintained for at least 5
-years). Hyperlinks may lead either to the actual attachment or to a valid
-containing "home page" for the resource.
+years.
 
-We prefer actual attachments over hyperlinks as we can serve them from the
-Anthology directly, but understand that some materials cannot fit within
-this size. Attachments do not need to be author-anonymized (although during
+Attachments do not need to be author-anonymized (although during
```
Comment (davidweichiang) [inline anthologize/README.md]: I don't understand this sentence at all.

File: anthologize/README.md (hunk 1 above)
Comment (mjpost) [reply anthologize/README.md]: I am actually thinking of removing step 3 and just doing that myself.

File: anthologize/README.md (hunk 2 above)
Comment (mjpost) [reply anthologize/README.md]: "Email a link to the file to the Anthology Director (please do not send the file as an email attachment)".

File: anthologize/README.md (hunk 1 above)
Comment (mjpost) [reply anthologize/README.md]: I want to give this some more thought, though, so I'm going to delay it a week till after the ACL deadline.

Comment (davidweichiang) [reply anthologize/README.md]: We can try to get the documentation in shape for the software as it stands, though, and if/when the software changes, the documentation should change with it.

File: anthologize/README.md (hunk 4 above)
Comment (davidweichiang) [reply anthologize/README.md]: I just deleted it.

**Pull request merged**